trillian_admin_api.proto

```diff
@@ -20,7 +20,7 @@ option java_outer_classname = "TrillianApiProto";
 
 package trillian;
 
-import "trillian.proto";
+import "github.com/google/trillian/trillian.proto";
```

Comment:
I don't like so much the idea of enforcing one proto file per package. I think I'd rather have the "github.com/" import in this case.

I can see how the "trillian/" import wouldn't work now (at least not in the current arrangement). In a 2nd attempt of mirroring googleapis, WDYT if we moved the proto files under a `proto` folder, like this:

```
github.com/google/trillian
  proto/
    trillian/
      crypto/
        sigpb/
          sigpb.proto
      trillian.proto
      trillian_admin_api.proto
      trillian_log_api.proto
      trillian_map_api.proto
```
It seems to me that, in this case, we could use `-I=$GOPATH/github.com/google/trillian/proto` and `import "trillian/trillian.proto"` as well. Mockgen commands would need to change a bit, but I think we could still output the .pb.go files to the root, as we do now.

@gdbelvin, thanks for the patience with my suggestions!